atrium/AtriumClient.php

```diff
@@ -247,6 +258,45 @@ function listAccountTransactions($userGUID, $accountGUID) {
     return $transactionArray;
   }
 
+  // ACCOUNT NUMBER
+
+  function listAccountAccountNumbers($userGUID, $accountGUID) {
+    $response = $this->makeRequest('GET', '/users/' . $userGUID . '/accounts/' . $accountGUID . '/account_numbers', '');
+    $accountNumbers = json_decode($response, true)['account_numbers'];
```

Comment:
Maybe it is better for `makeRequest` to decode the json for you? That way each caller doesn't have to decode it.